pkg/blobstore/replication/metrics_blob_replicator.go

```diff
@@ -0,0 +1,165 @@
+package replication
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/buildbarn/bb-storage/pkg/blobstore/buffer"
+	"github.com/buildbarn/bb-storage/pkg/blobstore/slicing"
+	"github.com/buildbarn/bb-storage/pkg/clock"
+	"github.com/buildbarn/bb-storage/pkg/digest"
+	"github.com/buildbarn/bb-storage/pkg/util"
+	"github.com/prometheus/client_golang/prometheus"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var (
+	replicatorOperationsPrometheusMetrics sync.Once
+
+	replicatorOperationsDurationSeconds = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: "buildbarn",
+			Subsystem: "blobstore",
+			Name:      "replicator_operations_duration_seconds",
```

Comment:
`blob_replicator_operations*` for consistency with `blob_access_operations_*` ?

Reply:
done.